src/core/mantle/evaluator/simple/index.ts

```diff
@@ -32,17 +36,26 @@ export function evalLayerAppearances(
       k,
       Object.fromEntries(
         Object.entries(v).map(([k, v]) => {
-          if (typeof v === "object" && "conditions" in v) {
-            return [k, evalExpression(v, layer, feature)];
-          }
-          return [k, v];
+          return [k, evalExpression(v, layer, feature)];
         }),
       ),
     ]),
   );
 }
 
-function evalExpression(_e: Expression, _layer: LayerSimple, _feature?: Feature): unknown {
-  // TODO: eval
-  return undefined;
+function evalExpression(
+  _styleExpression: StyleExpression,
+  _layer: LayerSimple,
+  _feature?: Feature,
```

Comment:
```suggestion
  styleExpression: StyleExpression,
  layer: LayerSimple,
  feature?: Feature,
```

I use `_` to supress "unused arguments" warning from TypeScript, but they are used.